examples/example2/CMakeLists.txt

```diff
@@ -23,28 +23,27 @@
 #
 #=============================================================================
 
-#AM_CPPFLAGS = -I$(top_srcdir)/fix-include -I$(top_srcdir)/include -DSRCDIR='"$(abs_srcdir)"'
-add_definitions("-DSRCDIR=\"${CMAKE_CURRENT_SOURCE_DIR}\"")
-
-# example1_CFLAGS = @OPENCL_CFLAGS@
-#set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -std=c99 ${OPENCL_CFLAGS}")
 add_compile_options(${OPENCL_CFLAGS})
 
-
 if (MSVC)
   set_source_files_properties( example2.c PROPERTIES LANGUAGE CXX )
 endif(MSVC)
 add_executable("example2" example2.c example2.cl)
 
-# example1_LDADD = @OPENCL_LIBS@ ../../lib/poclu/libpoclu.la
 target_link_libraries("example2" ${POCLU_LINK_OPTIONS})
 
-add_test("examples/example2_matrix_transpose" "example2")
+add_test("examples/example2" "example2")
+
+if (ENABLE_SPIR AND NOT (KERNELLIB_HOST_DISTRO_VARIANTS))
```

Comment:
Why is "KERNELLIB_HOST_DISTRO_VARIANTS"? Do you mean that SPIR is not functioning if host distro variants are built of the kernel lib? Why? If so, then would it be correct to disallow ENABLE_SPIR in case that is set?

Reply:
Hmm you're right, i should disable SPIR for distro builds for the time being. I haven't figured out yet why, but one of the LLVM passes wreaks havoc on SPIR-turned-to-LLVM-IR binaries.